static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -512,10 +517,14 @@ $navigation-horizontal-padding: $gs-gutter * 2;
         left: $navigation-horizontal-padding;
         right: 0;
         height: 1px;
-        background: darken($guardian-brand, 3%);
+        background: #003b5e;
     }
 }
 
+.navigation-no-border {
```

Comment:
I wouldn't expect this class name to give me `padding-bottom`. Can we rename this to be more clear?